guides/common/modules/proc_enrolling-projectserver-in-freeipa-domain.adoc

```diff
@@ -0,0 +1,121 @@
+====
+ifdef::foreman-deb[]
++
+. Ensure that the hostname is set to the fully qualified domain name (FQDN); the short name is not sufficient:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# hostname
+{foreman-example-com}
+----
++
+Otherwise, `{foreman-installer}` cannot generate the right principal name that is needed to join the realm.
+endif::[]
+
+.Verification
+* On your {ProjectServer}, check that you are able to resolve a user defined on the {FreeIPA} server.
+For example, to check the `admin` user that {FreeIPA} creates by default:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ id admin
+----
+
+.Enrolling a {ProjectServer} system as a {FreeIPA} client from the command line by using a one-time password
+====
+On the {FreeIPA} server, a user named _admin_ who has administrative privileges on the {FreeIPA} server prepares a host entry for the {ProjectServer} system:
+
+. Authenticate as the {FreeIPA} _admin_ user:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# kinit _admin_
+----
+. Optional: Verify that you have authenticated successfully:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# klist
+----
+. Create a host entry from the command line.
+Specify that you want to use a random password for the enrollment.
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# ipa host-add --random _{project-context}-server.example.com_
+--------------------------------------------------
+ Added host "{project-context}-server.example.com"
+ --------------------------------------------------
+  Host name: {project-context}-server.example.com
+  Random password: W5YpARl=7M.n
+  Password: True
+  Keytab: False
+  Managed by: ipa-server.example.com
+----
+. Enable access to the keytab file by creating a service principal for your {ProjectServer}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# ipa service-add HTTP/_{project-context}-server.example.com_
+----
+
+On the {ProjectServer} system, a user with {Project} administrative privileges enrolls the system into the {FreeIPA} domain:
+
+. Install the {FreeIPA} client packages:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {project-package-install} ipa-client
+----
++
+WARNING: The {project-package-install} command might restart {Project} services during the installation of the package.
+ifdef::satellite[]
+For more information about installing and updating packages on {Project}, see {AdministeringDocURL}Managing_Packages_on_the_Base_Operating_System_admin[Managing packages on the base operating system of {ProjectServer} or {SmartProxyServer}] in _{AdministeringDocTitle}_.
+endif::[]
+. Configure the {ProjectServer} system a client in {FreeIPA} by using the random password produced by `ipa host-add` in a previous step:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# ipa-client-install --password 'W5YpARl=7M.n'
```

Comment:
```suggestion
# ipa-client-install --password '_My_Password_'
```

Reply:
No change here. This is not a password set by user (so "my password" would be misleading). Also, it's the same example password that is used in [RHEL IdM docs](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html-single/installing_identity_management/index#proc_installing-a-client-by-using-a-one-time-password-interactive-installation_assembly_installing-an-idm-client).